modules/process/manager.go

```diff
@@ -6,13 +6,12 @@ package process
 
 import (
 	"bytes"
+	"context"
```

Comment:
We can vendor the package but the reason is less compelling since it doesn't affect end-users. Otherwise we can postpone the PR until Ubuntu catching up with the latest.

By the way, what keeps you (@bkcsoft @strk) using Ubuntu's Go rather than the official one? I'm curious if there's anything I could help.